app/controllers/claims_controller.rb

```diff
@@ -43,6 +43,11 @@ def start_new
 
   def redirect_to_existing_claim_journey
     new_journey = Journeys.for_policy(current_claim.policy)
+
+    # Set the params[:journey] to the new journey routing name so things like
+    # journey_session that rely on the journey param find the correct journey.
+    params[:journey] = new_journey::ROUTING_NAME
```

Comment:
Annoying but we build a slug sequence for a _different_ journey to the journey specified in the params which was breaking as we were passing the "wrong" answers to the slug sequence. This is ugly but I think there's a deeper problem with how this flow works.